playwright/tests/integration/mock/browser/tableCol.spec.ts

```diff
@@ -56,19 +56,20 @@ test('browser add table column doubleclick', async ({ page }) => {
 
 async function applyTableColBrowser(page: Page, expectedSelection: string = '', rowToCheck: number, numberOfRows?: number, dblClick?: boolean) {
   await browserBtn(page).nth(0).click();
-  await expect(page.getByRole('dialog')).toBeVisible();
-  await page.getByText('Table Column').first().click();
+  const browserDialog = page.getByRole('dialog');
```

Comment:
@ivy-lli I believe the only side effect I could identify from setting modal=false in our tests is that I now had to specifically target the dialog, as page.getByRole('row') was also picking up rows from behind the browser dialog. However, I don't think this is particularly concerning, do you?